File: src/cpp/count_space.c
#include <stdio.h>

int count_space(char str[]){
    int i, count = 0;
    for (i = 0; i < strlen(str); i++)
        if (str[i] == ' ')
            count++;
    return count;
}

int main(){
    printf("%d\n", count_space("This is a pen."));
    return 0;
}

File: src/CountSpace.java
class CountSpace {
    private int countSpace(String str){
        int count = 0;
        for (int i = 0; i < str.length(); i++)
            if (str.charAt(i) == ' ')
                count++;
        return count;
    }

    public static void main(String args[]){
        CountSpace cs = new CountSpace();
        System.out.println(cs.countSpace("This is a pen."));
    }
}
